src/sentry/workflow_engine/handlers/action/notification/handler.py

```diff
@@ -54,7 +54,23 @@ def handle_workflow_action(job: WorkflowJob, action: Action, detector: Detector)
 @action_handler_registry.register(Action.Type.WEBHOOK)
 @action_handler_registry.register(Action.Type.PLUGIN)
 class NotificationActionHandler(ActionHandler):
-    config_schema = {}
+    config_schema = {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "description": "The configration schema for a Notification Action",
+        "type": "object",
+        "properties": {
+            "target_identifier": {
+                "type": ["string", "null"],
+            },
+            "target_display": {
+                "type": ["string", "null"],
+            },
+            "target_type": {
+                "type": ["integer", "null"],
+                "enum": [at.value for at in ActionTarget] + [None],
```

Comment:
does this work 🤔 

```suggestion
                "enum": [*ActionTarget] + [None],
```

Reply:
seems like the splat works even though it's not just the values 🎉